R/create_round.R

```diff
@@ -123,20 +123,8 @@ create_round <- function(round_id_from_variable,
 
   property_names <- names(properties)
 
-
-  purrr::walk(
-    property_names[!property_names %in% c("submissions_due", "model_tasks")],
-    function(.x) {
-      check_input(
-        input = properties[[.x]],
-        property = .x,
-        round_schema,
-        parent_property = NULL,
-        scalar = TRUE,
-        call = call
-      )
-    }
-  )
+  check_properties_scalar(properties, round_schema, call = call)
+  check_properties_array(properties, round_schema, call = call)
```

Comment:
Adding this now allows for array arguments (which `derived_task_ids` is) to be validated when introduced in the next PR